src/js/components/award/shared/activity/ContractGrantActivityChart.jsx

```diff
@@ -78,13 +72,13 @@ const ContractGrantsActivityChart = ({
     const createYDomain = useCallback(() => {
         const clonedTransactions = cloneDeep(transactions);
```

Comment:
If there were any case, this would be it. Transactions is stored in state and used to draw the chart.  The container passes them to this chart sorted by action_date.  Array.sort() modifies the array and so does Array.pop()